index.html

```diff
@@ -8,10 +8,49 @@
 
     <!-- Link to main css -->
     <link rel="stylesheet" href="./assets/css/style.css">
+
+    <!-- Google font -->
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Poppins:wght@200;600&display=swap" rel="stylesheet">
 </head>
 <body>
-    <h1 class="header">
-        Hello Microverse!
-    </h1>
+    <header>
+        <div id="logo-div">
+            <p id="logo">My Logo</p>
+        </div>
+        <nav id="nav">
+            <img id="logo-icon" src="./assets/image/menu_icon.png" alt="menu icon">
+        </nav>
+    </header>
+
+    <section class="section">
+        <title id="title">
+            I’m Alexandra. Glad to see you! 
+        </title>
```

Comment:
> - [ ] I suggest you use an `h1` tag instead of the `title` tag. This is for Google and other search engines to know about a specific topic. You can [Check this link](https://www.reliablesoft.net/h1-tag/) for more info.☺